RUFAS/routines/manure/manure_treatments/base_manure_treatment.py

```diff
@@ -273,6 +275,22 @@ def _daily_update_helper(self) -> ManureTreatmentDailyOutput:
         """
         pass
 
+    @staticmethod
+    def _determine_outdoor_storage_temperature(air_temperature: float) -> float:
+        """Determines the temperature of the manure storage for outdoor storages.
```

Comment:
```suggestion
        """Determines the temperature of the manure storage for outdoor liquid and slurry storages.
```
Would this be more accurate? It seems that this method is not being used for solid outdoor storages

Reply:
Also, is this the temperature of the manure in the storage or is it the temperature of the air that is contained within the storage?

Reply:
I agree with Ed's comment that this method applies presently to liquid manure, and I think it's unlikely to be also applied to solid manure at some point (i.e. I think solid manure may use a different method), so let's go ahead and specify.
This is the temperature of manure _in_ storage - we should specify that as well.

Reply:
Thanks, Elle, what do you think about:

`Determines the temperature of the manure in outdoor liquid and slurry storages.`

vs.

`Determines the temperature of the liquid manure in outdoor storages.`

vs.

`Determines the temperature of the liquid manure in outdoor liquid and slurry storages.`

Reply:
I think option 1 is the best! This method applies to both liquid and slurry manure (although as you probably know, they're not really being formally differentiated in RuFaS right now. The distinction is still helpful though.)